src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Logging/IProjectLogger.cs

```diff
@@ -14,18 +12,5 @@ internal interface IProjectLogger
         ///     to the log.
         /// </summary>
         void WriteLine(string text);
-
-        /// <summary>
-        ///     Writes the text representation of the specified array of objects, 
-        ///     followed by the current line terminator, to the log using the 
-        ///     specified format information.
-        /// </summary>
-        /// <exception cref="ArgumentNullException">
-        ///     <paramref name="format"/> is <see langword="null"/>.
-        /// </exception>
-        /// <exception cref="FormatException">
-        ///     The format specification in <paramref name="format"/> is invalid.
-        /// </exception>
-        void WriteLine(string format, params object[] arguments);
     }
```

Comment:
I see why you removed this - you used interpolated strings, this is very deliberate that I did not use them, they allocate on every usage, even when logging is turned off.